Codeinwp/templates-patterns-collection
# New onboarding

### Summary
- Added the new Onboarding for Neve. 
- The new onboarding works only for new users of Neve.
- It redirects you automatically when you activate this plugin and if you have Neve active.
- If you install Neve after, you will see a submenu under Neve options named Onboarding but it will not start automatically

### Test instructions
- There are still some issues but I think we can start testing since this is a big project.
- I've mentioned the current known issues here: https://github.com/Codeinwp/neve-pro-addon/issues/2516#issuecomment-1706334460, as soon as I get some answers there, I will fix them
- The tracking part is not implemented yet
- Please note that the task mentioned that we'll not bother with the mobile version for now so please test only on desktop

**Note for the testers**: I've fixed some displaying conditions and now the onboarding lunches if:
- You are a new user ( you activated neve in the last day )
- You didn't finished the import before ( you didn't click on Subscribe /  Skip and view website )
- You activate Neve and have TPC installed and the above are true
- You activate TPC and have Neve installed and the above are true


Closes Codeinwp/neve-pro-addon#2516.
